src/main/java/uk/gov/justice/digital/service/operationaldatastore/dataaccess/OperationalDataStoreConnectionDetailsService.java

```diff
@@ -1,56 +0,0 @@
-package uk.gov.justice.digital.service.operationaldatastore.dataaccess;
-
-
-import io.micronaut.context.annotation.Requires;
-import jakarta.inject.Inject;
-import jakarta.inject.Singleton;
-import lombok.val;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import uk.gov.justice.digital.client.glue.GlueClient;
-import uk.gov.justice.digital.client.secretsmanager.SecretsManagerClient;
-import uk.gov.justice.digital.config.JobArguments;
-import uk.gov.justice.digital.datahub.model.OperationalDataStoreConnectionDetails;
-import uk.gov.justice.digital.datahub.model.OperationalDataStoreCredentials;
-
-import java.util.Map;
-
-/**
- * Responsible for retrieving details for connecting to the Operational DataStore.
- */
-@Singleton
-@Requires(property = "dpr.operational.data.store.write.enabled")
-public class OperationalDataStoreConnectionDetailsService {
```

Comment:
This was actually renamed to JDBCGlueConnectonDetailsService and slightly modified